readme.md

```diff
@@ -0,0 +1,206 @@
+# unist-lsp
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Size][size-badge]][size]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+Utility to convert between [unist][] and  [language server protocol][].
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`unistPointToLspPosition(point)`](#unistpointtolsppositionpoint)
+    *   [`unist-lsp(position)`](#unist-lspposition)
+    *   [`unistPositionToLspRange(position)`](#unistpositiontolsprangeposition)
+    *   [`lspRangeToUnistPosition(range)`](#lsprangetounistpositionrange)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Related](#related)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This package is a utility that takes converts between [unist][] and
+[language server protocol][].
+
+## When should I use this?
+
+This project is useful when you want to deal with unist ASTs and a language
+server.
+
+## Install
+
+This package is [ESM only][esm].
+In Node.js (version 14.14+, or 16.0+), install with [npm][]:
+
+```sh
+npm install unist-lsp
+```
+
+In Deno with [`esm.sh`][esmsh]:
+
+```js
+import {
+  lspPositionToUnistPoint,
+  lspRangeToUnistPosition,
+  unistPointToLspPosition,
+  unistPositionToLspRange,
+} from "https://esm.sh/unist-lsp@?0"
+```
+
+In browsers with [`esm.sh`][esmsh]:
+
+```html
+<script type="module">
+  import {
+    lspPositionToUnistPoint,
+    lspRangeToUnistPosition,
+    unistPointToLspPosition,
+    unistPositionToLspRange,
+  } from "https://esm.sh/unist-lsp@?0"
+</script>
+```
+
+## Use
+
+Say we have the following `example.md`:
+
+```markdown
+## Hello **World**!
+```
+
+…and next to it a module `example.js`:
+
+```js
+import {promises as fs} from 'node:fs'
+import {fromMarkdown} from 'mdast-util-from-markdown'
+
+
+const markdown = String(await fs.readFile('example.md'))
+const mdast = fromMarkdown(markdown)
+
+const range = unistPositionToLspRange(mdast.position)
+
+console.dir(range)
```

Comment:
```suggestion
console.log(range)
```

`log` should be fine here.
You could also show a couple of other examples, e.g., `fromPoint(tree.position.start)`. And the inverse